wouso/games/teamquest/utils/sample_team_quest.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env/python
+
+# To test, run from wouso folder using commands such as:
+# PYTHONPATH=../:. python games/teamquest/utils/create_sample_team_quest.py
+
+import argparse
+import sys
+import os
+import codecs
+import datetime
+
+# Setup Django environment.
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "wouso.settings")
+
+from django.contrib.auth.models import User
+from wouso.core.user.models import Race
+from wouso.core.user.models import Player
+from wouso.games.teamquest.models import TeamQuest, TeamQuestLevel, TeamQuestStatus, TeamQuestUser, TeamQuestGroup, TeamQuestLevelStatus, TeamQuestQuestion
+from wouso.core.qpool.models import Question, Answer, Category
+
+
+def create():
+    owner = User.objects.create(username='_sample_teamquest_user')
+    owner.set_password('password')
+    owner.save()
+    owner = owner.get_profile().get_extension(TeamQuestUser)
+    group = TeamQuestGroup.create(group_owner=owner, name='_sample_teamquest_group')
+    category = Category.add('teamquest')
+
+    number_of_levels = 5
+    questions = []
+    for index in range(number_of_levels * (number_of_levels + 1) / 2):
+        question = Question.objects.create(text='question'+str(index+1), answer_type='F',
+                category=category, active=True)
+        questions.append(question)
+        answer = Answer.objects.create(text='answer'+str(index+1), correct=True, question=question)
+
+    levels = []
+    # The start index of the questions sequence that goes in a level
+    base = 0
+    for index in range(number_of_levels):
+        level = TeamQuestLevel.create(quest=None, bonus=0,
+                questions=questions[base:base+number_of_levels-index])
+        levels.append(level)
+        base += number_of_levels - index
+
+    quest = TeamQuest.create(title="Sample Team Quest", start_time=datetime.datetime.now(),
+            end_time=datetime.datetime(2030,12,25), levels=levels)
+
+
+def status():
```

Comment:
Status function should be replaced by checking the objects' existance in the create function.